packages/frontend/core/src/blocksuite/initialization/index.ts

```diff
@@ -12,6 +12,14 @@ export interface DocProps {
   surface?: Partial<SurfaceBlockProps>;
   note?: Partial<NoteProps>;
   paragraph?: Partial<ParagraphProps>;
+  onLoad?: (
```

Comment:
onCreated?

Reply:
The naming is to align with the `load` callback in `Store`. Maybe change it to `onStoreLoad`?